atm.js

```diff
@@ -1,21 +1,65 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+// Set variables for checking and saving
+var cBalance = $(".balance").eq(0);
+cBalance.html("$0");
+var sBalance = $(".balance").eq(1);
+sBalance.html("$0");
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+// Set variables for input fields
+var cInput = $("#cInput");
+var sInput = $("#sInput");
 
-}
+$("#cDeposit").click(function (e){
+	e.preventDefault();
+	var deposit = parseFloat( $("#cInput").val() );
+	var total = parseInt(cBalance.text().replace("$", ""));
+	var newTotal = "$" + (total + deposit);
+	cBalance.html(newTotal);
+})
+// Withdraw money from checking, with overdraft protection
+$("#cWithdraw").click(function (e){
+	e.preventDefault();
+	var withdraw = parseFloat( $("#cInput").val() );
+	var cTotal = parseInt(cBalance.text().replace("$", ""));
+	var sTotal = parseInt(sBalance.text().replace("$", ""));
+	var combineTotal = cTotal + sTotal;
+	if(combineTotal - withdraw >= 0) {
+		if(cTotal - withdraw >= 0) {
+			console.log("Withdraw from checking!");
+			newTotal = "$" + (cTotal - withdraw);
+			cBalance.html(newTotal);
+		} else {
+			console.log("Withdraw from savings, overdraft protection!!");
+			sBalance.html( sTotal - (withdraw - cTotal) );
+			cBalance.html( "$0" );
+		}
+	} else {
+		console.log("Can't withdraw, bitch!");
+	}
+})
 
-function checkingWithdrawal(amount){
+$("#sDeposit").click(function (e){
```

Comment:
This has a lot of duplication with the `cDeposit` section above. Any way you could extract this out into a general function that could take an input to determine whether to act on checking or savings?